device/thunder_ripple_sdk/src/lib.rs

```diff
@@ -67,4 +67,6 @@ pub mod tests {
         pub mod thunder_wifi_pacts;
         pub mod thunder_window_manager_pacts;
     }
+    pub mod mock_extension_client;
```

Comment:
Can we build flag #[cfg(test)] this for a test utility?

Reply:
it is, it is on the mod that this is under